app/assets/stylesheets/welcome.css.scss

```diff
@@ -0,0 +1,26 @@
+#banner {
+  background-image: url(image_path("welcome.jpg"));
+  background-position: top center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: table;
+  min-height: 500px;
+  width: 100%;
+  .banner_content {
+		display: table-cell;
+		text-align: center;
+		vertical-align: middle;
+		h1 {
+			color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.